[2025-01-31T10:47:01.121+0000] {processor.py:186} INFO - Started process (PID=270) to work on /opt/airflow/dags/operators/testdag.py
[2025-01-31T10:47:01.121+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/operators/testdag.py for tasks to queue
[2025-01-31T10:47:01.123+0000] {logging_mixin.py:190} INFO - [2025-01-31T10:47:01.122+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/operators/testdag.py
[2025-01-31T10:47:01.138+0000] {logging_mixin.py:190} INFO - [2025-01-31T10:47:01.134+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/operators/testdag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/operators/testdag.py", line 7, in <module>
    from src.api_client.openweathermap_API import get_all_cities_air_quality
ModuleNotFoundError: No module named 'src'
[2025-01-31T10:47:01.140+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/operators/testdag.py
[2025-01-31T10:47:01.156+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/operators/testdag.py took 0.040 seconds
